src/ocaml_specific.ml

```diff
@@ -456,7 +457,36 @@ rule "ocamldoc: document ocaml project odocl & *odoc -> latex|dot..."
         The target foo.docdir/bar.dot will generate a dot graph of module \
         dependencies."
   (Ocaml_tools.document_ocaml_project
-      ~ocamldoc:Ocaml_tools.ocamldoc_l_file "%(dir).odocl" "%(dir).docdir/%(file)" "%(dir).docdir");;
+      ~tags:["texdoc"]
+      ~ocamldoc:Ocaml_tools.ocamldoc_l_file
+        "%(dir).odocl" "%(dir).docdir/%(file).tex" "%(dir).docdir");;
+
+rule "ocamldoc: document ocaml project odocl & *odoc -> latex (.ltx)"
+  ~prod:"%(dir).docdir/%(file).ltx"
+  ~dep:"%(dir).odocl"
```

Comment:
wouldn't it be possible to factorize the `.tex` and `.ltx` rules? From the top of my head I don't remember the kind of glob patterns we have available.